Texassemble/texassemble.cpp

```diff
@@ -2240,6 +2279,210 @@ int __cdecl wmain(_In_ int argc, _In_z_count_(argc) wchar_t* argv[])
             }
             break;
         }
+
+    case CMD_CUBE_FROM_HC:
+    case CMD_CUBE_FROM_VC:
+    case CMD_CUBE_FROM_VC_FNZ:
+    case CMD_CUBE_FROM_HT:
+    case CMD_CUBE_FROM_HS:
+    case CMD_CUBE_FROM_VS:
+        {
+            auto src = loadedImages.cbegin();
+            auto img = (*src)->GetImage(0, 0, 0);
+            size_t ratio_w = 1;
+            size_t ratio_h = 1;
+
+            switch (dwCommand)
+            {
+            case CMD_CUBE_FROM_HC:
+            case CMD_CUBE_FROM_HT:
+                ratio_w = 4;
+                ratio_h = 3;
+                break;
+
+            case CMD_CUBE_FROM_VC:
+            case CMD_CUBE_FROM_VC_FNZ:
+                ratio_w = 3;
+                ratio_h = 4;
+                break;
+
+            case CMD_CUBE_FROM_HS:
+                ratio_w = 6;
+                break;
+
+            case CMD_CUBE_FROM_VS:
+                ratio_h = 6;
+                break;
+
+            default:
+                break;
+            }
+
+            size_t twidth = width / ratio_w;
+            size_t theight = height / ratio_h;
+
+            if (((width % ratio_w) != 0) || ((height % ratio_h) != 0) || (twidth != theight))
```

Comment:
I don't think the result is required to be square.

Reply:
No problem removing the last condition because it's just a warning.
But it's not expected ratio when the result is not square, I think.

For example, it'll allow 40:3 aspect ratio for h-cross layout.